### PR TITLE
Extract and index full text content from PPLs

### PR DESCRIPTION
**Pull request by lennym**
When indexing PPLs create a second index for `projects-content` which also includes full text representations of the PPL sections.

This is created in a separate index so that it can be safely deployed to production without bringing down the existing projects index and interfering with daya-to-day business. It may be eventually desirable to consolidate into the main projects index.

### CODE REVIEW

File: lib/indexers/projects-content/extract-content.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
+require('@asl/service/lib/register');
```
Comment (wheelsandcogs) [inline lib/indexers/projects-content/extract-content.js:1]: What does this do?

Comment (lennym) [reply lib/indexers/projects-content/extract-content.js:1]: It includes a pre-configured implementation of babel-register which rewires `require` to also do a babel transpile on any files it `require`s.

Specifically in this case it allows us to load the schema files from `@asl/projects` without getting parsing errors when node finds the `import` statements in those files.

File: lib/indexers/projects-content/index.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+const indexProject = (esClient, project, ProjectVersion) => {
+  return ProjectVersion.query()
+    .where({
+      // look for most recent submitted draft for inactive projects
+      status: project.status === 'inactive' ? 'submitted' : 'granted',
+      projectId: project.id
+    })
+    .orderBy('updatedAt', 'desc')
+    .first()
```
Comment (wheelsandcogs) [inline lib/indexers/projects-content/index.js:21]:
```suggestion
    .limit(1)
    .first()
```

Comment (wheelsandcogs) [reply lib/indexers/projects-content/index.js:21]: As far as I can tell from the docs, Objection's `first()` does not apply a limit, so all the records are returned and then it's just an array op.

File: lib/indexers/projects-content/extract-content.js (hunk 1 above)
Comment (wheelsandcogs) [reply lib/indexers/projects-content/extract-content.js:1]: 👍

**Pull request merged**